{"frames":[{"func":"df_1 %>% tidyr::pivot_longer(names_to = \"time\", cols = -subject)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"tidyr::pivot_longer(., names_to = \"time\", cols = -subject)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"에러: 객체 'df_1'를 찾을 수 없습니다\n"}